packages/experiments/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -0,0 +1,113 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
What is this example for?

Reply:
This example was not finished thus not used in the example page. Should be good with the next push.

Reply:
This example was not ready at the time of submission thus was not included in the example page. Will add it with the next push.